README.md

```diff
@@ -1,37 +1,37 @@
+- [My Account: Returns / Exchange](http://commercetools.github.io/commercetools-sunrise-theme/site/en/my-account-returns-exchange.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/my-account-returns-exchange.html) )
+- [My Account: Returns / Exchange - Single Return](http://commercetools.github.io/commercetools-sunrise-theme/site/en/my-account-returns-exchange-order.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/my-account-returns-exchange-order.html) )
+- [My Account: Wishlist](http://commercetools.github.io/commercetools-sunrise-theme/site/en/my-account-wishlist.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/my-account-wishlist.html) )
+- [My Account: Login](http://commercetools.github.io/commercetools-sunrise-theme/site/en/my-account-login.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/my-account-login.html) )
+- [Confirmation Email](http://commercetools.github.io/commercetools-sunrise-theme/site/en/confirmation-email.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/confirmation-email.html) )
+- [Dispatch Email](http://commercetools.github.io/commercetools-sunrise-theme/site/en/dispatch-email.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/dispatch-email.html) )
+- [Registration Email](http://commercetools.github.io/commercetools-sunrise-theme/site/en/registration-email.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/registration-email.html) )
+- [Password Reset Email](http://commercetools.github.io/commercetools-sunrise-theme/site/en/password-reset-email.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/password-reset-email.html) )
+- [No Search Result](http://commercetools.github.io/commercetools-sunrise-theme/site/en/no-search-result.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/no-search-result.html) )
+- [Mix & Match](http://commercetools.github.io/commercetools-sunrise-theme/site/en/mix-match.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/mix-match.html) )
+- [Contact Form](http://commercetools.github.io/commercetools-sunrise-theme/site/en/contact-form.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/contact-form.html) )
+- [Store Finder](http://commercetools.github.io/commercetools-sunrise-theme/site/en/store-finder.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/store-finder.html) )
+- [FAQ Page](http://commercetools.github.io/commercetools-sunrise-theme/site/en/faq.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/faq.html) )
+- [Imprint Page](http://commercetools.github.io/commercetools-sunrise-theme/site/en/imprint.html) ( [DE](http://commercetools.github.io/commercetools-sunrise-theme/site/de/imprint.html) )
```

Comment:
That's the reason why I wanted to create the autogenerated HTML or MD with all the files :(